libs/services/src/SlippageService/SlippageServiceMain.spec.ts

```diff
@@ -1,47 +1,445 @@
-import { Container, injectable } from 'inversify';
-import { SlippageService, slippageServiceSymbol } from './SlippageService';
 import { SlippageServiceMain } from './SlippageServiceMain';
 import {
   PricePoint,
-  PriceStrategy,
   SupportedChainId,
   UsdRepository,
-  usdRepositorySymbol,
 } from '@cowprotocol/repositories';
+import ms from 'ms';
 
-@injectable()
-class UsdRepositoryMock implements UsdRepository {
-  getUsdPrice(
-    chainId: SupportedChainId,
-    tokenAddress: string
-  ): Promise<number | null> {
-    throw null;
-  }
-  getUsdPrices(
-    chainId: SupportedChainId,
-    tokenAddress: string,
-    priceStrategy: PriceStrategy
-  ): Promise<PricePoint[] | null> {
-    throw null;
-  }
-}
+const MIN_SLIPPAGE = 50;
+const MAX_SLIPPAGE = 200;
+
+const FIVE_MIN = ms('5min');
+const FOUR_MIN = ms('4min');
+const SIX_MIN = ms('6min');
+
+const getUsdPrice = jest.fn();
+const getUsdPrices = jest.fn();
+
+const POINTS_VOLATILITY_ZERO = getPoints([100, 100, 100, 100]);
+const POINTS_WITH_HIGH_VOLATILITY = getPoints([100, 105, 110, 120]); // 10% each 5min
```

Comment:
First point was 5%. Adjusted to match comment.

```suggestion
const POINTS_WITH_HIGH_VOLATILITY = getPoints([100, 110, 120, 130]); // 10% each 5min
```